{{ cookiecutter.repo_name }}/tox.ini

```
@@ -1,12 +1,17 @@
 [tox]
 isolated_build = true
-envlist = clean,py37-django22,py37-django32,{38,39,310}-django{22,32,40},report
+envlist =
+    clean
+    py37-django{22,32}
+    py{38,39,310}-django{22,32,40}
```

next time use a suggestion?
```suggestion
    py{38,39,310}-django{22,32,40,41}
```